core/next.config.js

```diff
@@ -28,6 +28,17 @@ const nextConfig = {
       });
     }
 
+    // Limit the number of chunks to reduce CDN requests, which contribute to Edge Request costs
+    if (!isServer) {
+      config.plugins.push(
+        new (require('webpack').optimize.LimitChunkCountPlugin)({
+          maxChunks: process.env.WEBPACK_MAX_CHUNKS
+            ? parseInt(process.env.WEBPACK_MAX_CHUNKS, 5)
```

Comment:
It's pretty rare to see a base 5 number. Might want to make it base 10 🤔 
```suggestion
            ? parseInt(process.env.WEBPACK_MAX_CHUNKS, 10)
```